snap/integrity/integrity_test.go

```diff
@@ -218,6 +218,8 @@ func (s *IntegrityTestSuite) TestGenerateAndAppendSuccess(c *C) {
 
 	snapPath, _ := snaptest.MakeTestSnapInfoWithFiles(c, "name: foo\nversion: 1.0", nil, nil)
 
+	const verityHashSize = 8192
```

Comment:
nitpick: we could add a small comment explaining that this was empirically verified to be the size of the hash file with a minimal snap and it's hardcoded since there's no quick/simple way to calculate it (just so no one is left wondering where this is coming from)